### PR TITLE
Add missing mix-col-span mixin

### PR DESCRIPTION
**Pull request by michael-hamilton**
This should fix issue #19538

### CODE REVIEW

**Comment by cvrebert** [comment]
I'm not sure whether it's the CSS or the docs that need fixing here.
CC: @mdo for review

File: scss/mixins/_grid.scss [hunk 1]
```diff
@@ -51,6 +51,10 @@
   }
 }
 
+@mixin mak-col-span($size, $columns: $grid-columns) {
```
Comment (curtgrimes) [inline scss/mixins/_grid.scss]: @michael-hamilton `mak-col-span` should be `make-col-span`

Comment (michael-hamilton) [inline scss/mixins/_grid.scss]: Darn... not sure how that happened.  I'll fix it.

**Comment by mdo** [comment]
Thanks, but not the solution for that bug.

**Pull request closed without merging**